docs/community/SIG-Governance.md

```diff
@@ -0,0 +1,45 @@
+# OCM SIG Governance Model
+
+This document defines the governance framework for Special Interest Groups (SIGs) in the Open Component Model (OCM) project. It is designed to be lightweight and scalable for a small open source project, while ensuring transparency, consistency, and community alignment.
+
+## 1. Purpose
+
+SIGs are groups of contributors focused on specific technical or community topics within OCM. The governance model ensures clear roles, decision-making, and lifecycle management for all SIGs.
+
+## 2. Roles & Responsibilities
+
+- **Chair**: Facilitates meetings, represents the SIG, ensures process adherence. (At least one Chair is required.)
+- **Tech Lead**: Guides technical direction, reviews contributions, supports the Chair. (At least one Tech Lead is required.)
+- **Members**: Regular contributors with voting rights.
+
+## 3. Decision-Making
+
+- Prefer consensus; if consensus cannot be reached, a simple majority vote among Members decides.
```

Comment:
How is voting perceived? What are the interaction with LFX and the committee presentation. I think this needs some detailing